templates/react/src/config/index.js

```diff
@@ -0,0 +1,15 @@
+export default {
```

Comment:
this file is only used for local dev, see corresponding template file that is used in code generation